attributes/default.rb

```diff
@@ -13,3 +16,8 @@
 default['optoro_kafka']['disks'] = ['/dev/sdf']
 override['kafka']['server.properties']['log.dirs'] = '/kafka'
 default['optoro_kafka']['disk_size'] = 1024
+default['kafka']['server.properties']['kafka.metrics.reporters'] = 'com.criteo.kafka.KafkaGraphiteMetricsReporter'
+default['kafka']['server.properties']['kafka.graphite.metrics.host'] = 'localhost'
+default['kafka']['server.properties']['kafka.graphite.metrics.port'] = '6008'
+default['kafka']['server.properties']['kafka.graphite.metrics.group'] = node['fqdn']
```

Comment:
so in this case the output already includes 'kafka' because all of the mbeans are name spaced. BUT some of the jvm stuff isn't...
so we could either have:

kafka-001.kafka.server.foobar.metric
kafka-001.jvm.foobar.metric
or 

kafka-001.kafka.kafka.server.foobar.metric
kafka-001.kafka.jvm.foobar.metric

Which makes more sense?